.github/workflows/ci.yml

```diff
@@ -81,74 +89,94 @@ jobs:
             clang-runtime: '16'
             cling: Off
             cppyy: Off
-          - name: win2022-msvc-cling
+          - name: win2022-msvc-cling-18
+            os: windows-2022
+            compiler: msvc
+            clang-runtime: '18'
+            cling: On
+            cling-version: 'refs/heads/master'
+          - name: win2022-msvc-cling-16
             os: windows-2022
             compiler: msvc
-            clang-runtime: '13'
+            clang-runtime: '16'
             cling: On
-            cling-version: '1.0'
+            cling-version: 'v1.1'
             cppyy: Off
-          - name: osx14-arm-clang-clang-repl-19
+          - name: osx14-arm-clang-repl-19
```

Comment:
With the exception of Windows the names of jobs are currently
'operation system'-architechture-compiler-'cling or clang-repl version'

Reply:
I would prefer easier readability. Compiler backend are either clang-repl or cling. 'clang-clang-repl' does not make sense

Reply:
@aaronj0 That's fine to do but we need to be consistent, so you've need to drop the msvc and gcc12 from the other jobs for consistency.